app/routes.js

```diff
@@ -13,7 +13,8 @@ router.get('/', async (req, res) => {
 
 router.get('/view-solution/:solutionId', async (req, res) => {
   const { solutionId } = req.params;
-  const context = await getPublicSolutionById(solutionId);
+  const { browse: browseSolutionType } = req.query;
```

Comment:
I think something like `filterType` might be clearer. Since it's possible to view a foundation solution from `all`

Reply:
👍 done